src/OpenTelemetry.Exporter.OpenTelemetryProtocol/OtlpLogExporterHelperExtensions.cs

```diff
@@ -41,8 +49,22 @@ public static OpenTelemetryLoggerOptions AddOtlpExporter(this OpenTelemetryLogge
     /// <returns>The instance of <see cref="OpenTelemetryLoggerOptions"/> to chain the calls.</returns>
     public static OpenTelemetryLoggerOptions AddOtlpExporter(
         this OpenTelemetryLoggerOptions loggerOptions,
-        Action<OtlpExporterOptions> configure)
-        => AddOtlpExporterInternal(loggerOptions, configure);
+        Action<OtlpExporterOptions>? configure)
+    {
+        Guard.ThrowIfNull(loggerOptions);
+
+        var name = Options.DefaultName;
+
+        return loggerOptions.AddProcessor(sp =>
```

Comment:
From end user perspective: This scenario of enabling IConfiguration/SdkLimitOptions/BatchOptions will only be possible when they have the servicecollection already available like in ASP.NET Core apps. Unlike in case of tracing/metrics where they can call ConfigureServices extension. Is my understanding correct?

Reply:
Typically I would expect only host users (AspNetCore or generic host) to care about deep IConfiguration integration. But it is also available for manual scenarios too if users want it.

Here is how you would do it manually for traces & logs:

```csharp
var config = new ConfigurationBuilder()
    .AddJsonFile("appSettings.json")
    .AddEnvironmentVariables()
    .AddCommandLine(args)
    .Build();

using var tracerProvider = Sdk.CreateTracerProviderBuilder()
    .ConfigureServices(services => services.AddSingleton<IConfiguration>(config))
    .AddOtlpExporter()
    .Build();

using var loggerFactory = LoggerFactory.Create(builder =>
{
    builder.Services.AddSingleton<IConfiguration>(config);

    builder.AddOpenTelemetry(logging => logging.AddOtlpExporter());
});
```

Reply:
Could we add a test to validate this scenario?

Reply:
see: https://github.com/open-telemetry/opentelemetry-dotnet/blob/a505998afb07b339afa76d9f7f1b860941b315d4/test/OpenTelemetry.Exporter.OpenTelemetryProtocol.Tests/OtlpLogExporterTests.cs#L1297